gcn/notices/core/FollowUp.schema.json

```diff
@@ -19,8 +19,7 @@
     },
     "reference": {
       "type": "object",
-      "additionalProperties": { "type": "number" },
-      "description": "Reference as distributed by the notices or circulars or ATel, ex. gcn.notices.Fermi.gbm.alert: 4642"
+      "description": "Key-value pairs to reference notices, circulars, ATel, etc. For notices, the key should be the topic as a string, and the value should be the timestamp of the notice (ex: 'gcn.notices.glowbug.alert':'2023-01-01T03:24:36.0Z'). For circulars the key should be 'gcn.circulars' and the value should be the circular ID (ex: 'gcn.circular':32517)"
```

Comment:
```suggestion
      "description": "Key-value pairs to reference notices, circulars, ATel, etc. For notices, the key should be the topic as a string, and the value should be the timestamp of the notice (ex: 'gcn.notices.glowbug.alert':'2023-01-01T03:24:36.0Z'). For circulars the key should be 'gcn.circulars' and the value should be the circular ID (ex: 'gcn.circular':32517), similarly for ATel."
```

Reply:
```suggestion
      "description": "Key-value pair to reference notices, circulars, ATel, etc. that point to localization being followed up. For notices, the key can be the topic as a string with the value as the timestamp of the notice (ex: 'gcn.notices.glowbug.alert':'2023-01-01T03:24:36.0Z') or another string descriptor of the follow-up source. For Circulars or ATels, the key-value pair should be the system and ID (ex: 'circular':32517 or 'atel':12345)."
```

Reply:
similar, but I think we were using fields intended for the alert producer.  At the end of the day, we're in production now with:

"ref_ID": "MS230427r-1-Preliminary" type references.  which encodes all we need.  IT does not provide a good clean handle to link back to the original LVK notice, without some trolling on gracedb site.

Reply:
I see, so your setup should still work and this `reference`  update will just allow for any additional generic key-value pairs that might be useful in other cases

Reply:
correct.  I do think eventually there needs be a simple way to have a reference to notices from others that's easy to find, and makes tracking down the original alert possible.  I guess this will be more important for some notice visualization tool or website so it can make these connections.